CONTRIBUTING.md

```diff
@@ -1,3 +1,5 @@
+Contributions from young and old, experienced and inexperienced, are welcome!
```

Comment:
I think I would prefer this without the explicit reference to age, which is a bit odd to me.